core/domain/stats_domain.py

```diff
@@ -30,16 +30,27 @@ class StateRuleAnswerLog(object):
     specific storage model used.
     """
     def __init__(self, answers):
-        # This dict represents a log of answers that hit this rule and that
-        # have not been resolved. The keys of this dict are the answers encoded
-        # as HTML strings, and the values are integer counts representing how
-        # many times the answer has been entered.
+        """Initializes answers dictionary with answers that hit this rule
+        and have not been resolved.
+
+        Args:
+            answers: dict. The keys of the dict are the answers encoded as HTML
+                strings, and the values are integer counts representing how
+                many times the answer has been entered.
+        """
         self.answers = copy.deepcopy(answers)
 
     @property
     def total_answer_count(self):
-        """Total count of answers for this rule that have not been resolved."""
-        # TODO(sll): Cache this computed property.
+        """Total count of answers for this rule that have not been resolved.
+
+        Returns:
+            int. A int specifying the total number of answers for this rule that
+            have not been resolved.
+
+        Todo:
```

Comment:
These are not part of the docstring!

In a comment just below the docstring, add:

`TODO(sll): Cache this computed property.`

similar to how it was before. This is just a temporary TODO.